assembl/static2/webpack.config.js

```diff
@@ -132,9 +132,10 @@ module.exports = {
     },
     mode: 'production',
     optimization: {
-        minimizer: [
-            new UglifyJsPlugin({ sourceMap: true, parallel: true, cache: true })
-        ]
+      minimizer: [new TerserPlugin({
+        cache: true,
+        parallel: true,
```

Comment:
Why did you remove `sourceMap: true`?

Reply:
No good reason except it slows down the compilation. I'm in testing mode with this at the moment but I needed it in the develop branch for CI purposes. I need to also move the dep to devdep if we keep it. However I might just revert the commit if I don't see the improvements. At the moment it looks ×2 faster

Reply:
uglifyjs-webpack-plugin is unmaintained. Recent webpack version replaced uglifyjs-webpack-plugin  by terser-webpack-plugin. You should actually remove uglifyjs-webpack-plugin from the dependency, upgrade webpack, remove the the optimizer section in webpack.config.js.
I have the change in a old branch I never pushed. I can do it again monday if you want.
And you should commit the yarn.lock file when you add or update dependencies like that.